packages/moode-player/build.sh

```diff
@@ -134,44 +132,32 @@ fi
 
 rm -f "${BUILD_ROOT_DIR}/moode-sqlite3.db" || true
 # move it to the dist location
-mv -f ${BUILD_ROOT_DIR}/moode-stations-*_${PKGVERSION}.zip  "${BASE_DIR}/dist/binary/"
+mv -f "${BUILD_ROOT_DIR}"/moode-stations-*_"${PKGVERSION}.zip"  "${BASE_DIR}/dist/binary/"
 
 # location for files that should overwrite existing files (not owned by moode-player)
 NOT_OWNED_TEMP=${PKG_ROOT_DIR}/usr/share/moode-player
-mkdir -p "${NOT_OWNED_TEMP}"
-
-# /boot
-rsync -av --prune-empty-dirs --exclude *.sed* --exclude *.overwrite* --exclude *.ignore* $MOODE_DIR/boot/ $PKG_ROOT_DIR/boot/
```

Comment:
Sorry most of your refactoring is an improved, but I don't like this commit with compacting thethe rsync command.
Yes it uses less lines, but it far less clear (no overview). Readability wins for me

Reply:
let me try something else then :)

Reply:
would a formatting change help?
```
# rsync dist stuff from moode source to target build directory for packaging
rsync \
    -avR \
    --prune-empty-dirs \
    --exclude={'*.sed*','*.overwrite*','*.ignore*','moode-sqlite3.db','radio-logos'} \
    boot/ \
    etc/ \
    lib/ \
    var/ \
    "${PKG_ROOT_DIR}"
```

Reply:
as an alternative to that i would maybe suggest at least merging rsync commands that share the same exclude and include parameters into one

Reply:
Sorry I still like it better the original way, that is for both commits:
4a92044
7ff8995

The idea is to group the actions together that have to deal with the same subject; a certain directory or file.
In the new form the directories are grouped arround with the action instead.

This makes the mental model harder when maintaining the code. And remember this is just style issue ;-)
So please revert both commits.